conans/test/integration/command_v2/test_combined_pkglist_flows.py

```diff
@@ -168,6 +169,90 @@ def test_graph_pkg_list_only_built(self):
         assert len(pkglist["default"]) == 2
 
 
+class TestPkgListFindRemote:
+    """ we can recover a list of remotes for an already installed graph, for metadata download
+    """
+    def test_graph_2_pkg_list_remotes(self):
```

Comment:
Maybe 
```
diff --git a/conans/test/integration/command_v2/test_combined_pkglist_flows.py b/conans/test/integration/command_v2/test_combined_pkglist_flows.py
index e6c9ba955..fbd88317d 100644
--- a/conans/test/integration/command_v2/test_combined_pkglist_flows.py
+++ b/conans/test/integration/command_v2/test_combined_pkglist_flows.py
@@ -180,6 +180,7 @@ class TestPkgListFindRemote:
         c.run("create zlib")
         c.run("create app ")
         c.run("upload zlib* -c -r=default")
+        c.run("upload zlib* -c -r=remote2")
         c.run("upload app* -c -r=remote2")
 
         # This install, packages will be in the cache
@@ -197,8 +198,9 @@ class TestPkgListFindRemote:
         assert "Local Cache" not in pkglist
         assert len(pkglist["default"]) == 1
         assert "zlib/1.0" in pkglist["default"]
-        assert len(pkglist["remote2"]) == 1
+        assert len(pkglist["remote2"]) == 2
         assert "app/1.0" in pkglist["remote2"]
+        assert "zlib/1.0" in pkglist["remote2"]
```

to show that it has no problem handling cases where the lib is present in more than one site, even if the user later has to decide where to get the metadata from?

Reply:
Done